src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -103,8 +113,8 @@ object ExportVCF {
 
   def infoType(f: Field): String = {
     val tOption = f.typ match {
-      case TArray(elt, _) => infoType(elt)
-      case TSet(elt, _) => infoType(elt)
+      case TArray(elt, _) if !elt.isOfType(TBoolean()) => infoType(elt)
```

Comment:
You're already in a match, just do `TArray(TBoolean(_), _) => ...`.

Reply:
It's matching Arrays whose elements are NOT boolean. How about this?
```
    val tOption = f.typ match {
      case TArray(TBoolean(_), _) | TSet(TBoolean(_), _) => None
      case TArray(elt, _) => infoType(elt)
      case TSet(elt, _) => infoType(elt)
      case t => infoType(t)
    }
```